packages/forma-36-website/src/pages/guidelines/copy/suggested-reading/index.mdx

```diff
@@ -0,0 +1,19 @@
+# Suggested reading
+
+We’re grateful to teams that have shared their process and approach through their design systems and copy style guides.
```

Comment:
I also think this should be shared internally but not be part of our design system documentation.

Reply:
I initially suggested the same, but Sara won me over with the argument that this shows we're open to the community and are clear about our inspiration and aspirations. There's also a couple other examples of design systems using it. I don't have a strong opinion.

Reply:
I think it'd make sense to talk about these resources as part of a blog post, but I feel the design system itself should be focused on how to build a Contentful product, not how to build a design system.